backend/src/main/java/gov/cdc/usds/simplereport/service/model/SmsDeliveryResult.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.usds.simplereport.service.model;
+
+public class SmsDeliveryResult {
```

Comment:
discussion: Is there a better name for this class? I am simultaneously working on #1931 which gets adds a webhook for Twilio to inform us of the message status as it gets enqueued, sent, and received.

Reply:
Good call-out. Maybe something like `PatientLinkSmsDeliveryResult` sufficiently narrows the scope? It's a bit tricky to fully disambiguate since the class you'll be adding in #1931 could _also_ represent a "patient link SMS delivery result". What do you think?

Reply:
It's something like `SmsAPICallResult` I think. The word that I think should probably not be in the name is "Delivery" because the success of the API call doesn't really speak to the delivery status.

Reply:
Addressed in c5a4c27.